src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/list-items/list/list.component.html

```diff
@@ -1,13 +1,13 @@
 <div class="list-container">
   <div class="list-content" *ngIf="schema | async as schemaDetail">
-    <pp-list-browser 
-      [schema]="schemaDetail"
-      [searchString]="searchQuery | async"
-      [filter]="filter | async"
-      [enableSelection]="true"
-      [selectedItemIds]="selectedItemsIds"
-      (selectedItemsChange)="selectedItemsChange($event)">
-    </pp-list-browser>
+      <pp-list-browser *ngIf="schemaDetail" 
+        [schema]="schemaDetail"
+        [searchString]="(searchQuery | async) || ''"
```

Comment:
Do not use the OR string in the async pipes

Reply:
Created commit with all the changes needed to do it, but still think it would be better to maintain as is, since the possibility of it being null is not intended behavior and only due to the nature of the async pipe